rust/sealevel/environments/mainnet3/gas-oracle-configs.json

```diff
@@ -96,6 +96,22 @@
       },
       "overhead": 600000
     },
+    "superseed": {
+      "oracleConfig": {
+        "tokenExchangeRate": "239372224587870308412",
+        "gasPrice": "234340482",
+        "tokenDecimals": 18
+      },
+      "overhead": 166887
+    },
+    "ink": {
+      "oracleConfig": {
+        "tokenExchangeRate": "239372224587870308412",
+        "gasPrice": "234340482",
+        "tokenDecimals": 18
+      },
+      "overhead": 166887
+    },
```

Comment:
confirmed that this was the output from the `yarn tsx ./scripts/sealevel-helpers/print-gas-oracles.ts -e mainnet3` script in the runbook 👍